qt/Torrent.h

```diff
@@ -291,6 +291,16 @@ class Torrent : public QObject
         return piece_count_;
     }
 
+    std::vector<int> availability() const noexcept
```

Comment:
Here and below: needless copying (which may lead to allocation, which may in turn lead to exception violating `noexcept`), return `const&` (and save in `const&` variable at call site) instead.

Reply:
Returned by reference now but still unnecessarily copied in `TorrentDelegate::drawTorrent()`